html/pfappserver/root/static.alt/src/views/Nodes/_store/index.js

```diff
@@ -231,6 +231,19 @@ const actions = {
       })
     })
   },
+  registerBulkNodes: ({commit}, macs) => {
+    return new Promise((resolve, reject) => {
+      api.registerBulkNodes(macs).then(response => {
+        response.items.filter(item => item.status === 'success').forEach(function (item, index, items) {
+          commit('ITEM_UPDATED', { mac: item.mac, prop: 'status', data: 'reg' })
+        })
+        resolve(response)
+      }).catch(err => {
+        commit('ITEM_ERROR', err.response)
```

Comment:
This mutation doesn't exist. I guess you wanted to rename `NODE_ERROR`?

Reply:
I wanted to keep NODE_* and ITEM_* seperated, thus I created a ITEM_ERROR instead or reusing NODE_ERROR.

Reply:
Got it. Thanks.